sdks/python/apache_beam/io/mongodbio.py

```diff
@@ -192,7 +192,7 @@ def get_range_tracker(self, start_position, stop_position):
   def read(self, range_tracker):
     with MongoClient(self.uri, **self.spec) as client:
       all_filters = self._merge_id_filter(range_tracker)
-      docs_cursor = client[self.db][self.coll].find(filter=all_filters).sort("_id")
+      docs_cursor = client[self.db][self.coll].find(filter=all_filters).sort([('_id', ASCENDING)])
```

Comment:
Probably need to break into two lines since it is violating line length lint rule.